File: Entities/Response/PilotResponse.cs

namespace Entities.Response
{
    public class PilotResponse
    {
        public string? Id { get; set; }
        public string? Name { get; set; }
        public string? LastName { get; set; }
        public string? License { get; set; }
        public string? Phone { get; set; }
        public string? Email { get; set; }
        public int? State { get; set; } = 1;
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? CreatedBy { get; set; }
        public string? UpdatedBy { get; set; }
    }
}

File: dashboard_react.client/src/types/PilotResponse.ts
import { SearchCatalogue } from "./Catalogue";

export interface PilotResponse extends SearchCatalogue {
  id?: string;
  name: string;
  lastName: string;
  fullName: string;
  license: string;
  phone: string;
  email: string;
  state: number;
  createdAt: string | null;
  updatedAt?: string | null;
  createdBy: string | null;
  updatedBy?: string | null;
}
